batch/Dockerfile.worker

```diff
@@ -19,6 +20,14 @@ RUN echo "APT::Acquire::Retries \"5\";" > /etc/apt/apt.conf.d/80-retries && \
     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
     hail-apt-get-install fuse gcsfuse=0.41.12
 
+RUN curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg && \
```

Comment:
Can you add a comment explaining why the `nvidia-container-toolkit` is necessary here as well as on the host via the boot disk image?